test/generate_parse_fixture_yml.py

```diff
@@ -64,6 +64,7 @@ def distribute_work(work_items: List[S], work_fn: Callable[[S], None]) -> None:
                         print(f"  - {fname!r}")
 
     if errors:
+        print(errors)
```

Comment:
Locally I don't know why parsing failed without this when running `tox -e generate-fixture-yml -- --dialect bigquery`